auth/metrics.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 The etcd Authors
```

Comment:
Nit, this is a new file, we should have used year 2020 here (instead of 2015) for an easy future reference of when the file was created. Considering this is cherry pick, doing a separate PR in the master and then cherry pick it would be nice. Thanks!